deploy/skaffold/Dockerfile.deps.slim

```diff
@@ -17,9 +17,9 @@ ARG ARCH=amd64
 # Download kubectl
 FROM alpine:3.10 as download-kubectl
 ARG ARCH
-# Track default version installed by Google Cloud SDK: 406.0.0 moved to 1.22(.14)
+# Track default version installed by Google Cloud SDK: 406.0.0 moved to 1.23(.13)
```

Comment:
I think this should be 414.0.0? as in `Dockerfile.deps.lts`?

Reply:
resolved now, thanks!